app/javascript/sdk/bubbleHelpers.js

```diff
@@ -52,6 +52,7 @@ export const createBubbleIcon = ({ className, path, target }) => {
   }
 
   target.className = bubbleClassName;
+  target.title = $t('START_CONVERSATION');
```

Comment:
```suggestion
  target.title = 'Open chat window';
```

Reply:
@drexlma $t methods wouldn't work here as this piece of code is not written in Vue. It's vanilla JS. Let me know if the suggestion works.